packages/transfer-app/src/Transfer.tsx

```diff
@@ -2,20 +2,31 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
+import { Container, Grid } from '@polkadot/ui-components';
+
```

Comment:
I would propose to remove this line break everywhere, so that it's:
```
{absolute imports block}
line break
{relate imports block}
```

There are a lot of npm packages that start with @, I'm not sure it makes sense to create a block specifically for those ones.